pkg/Microsoft.Private.PackageBaseline/packageIndex.json

```diff
@@ -2227,12 +2227,14 @@
         "4.5.0",
         "4.5.1",
         "4.5.2",
-        "4.5.3"
+        "4.5.3",
+        "4.5.4"
       ],
       "InboxOn": {},
       "AssemblyVersionInPackageVersion": {
         "4.0.0.0": "4.5.1",
-        "4.0.0.1": "4.5.2"
+        "4.0.0.1": "4.5.2",
+        "4.0.0.2": "4.5.4"
```

Comment:
These changes should be reverted from 2.1

Reply:
This specific line only? And then don't change the assemblyVersion in dir.props?

Reply:
The entire change. The assemblyVersion also needs to be updated here.

Reply:
> These changes should be reverted from 2.1

> The assemblyVersion also needs to be updated here.

? I'm super lost

Reply:
sry the packageIndex.json shouldnt be updated in release/2.1 branch.
The assemblyVersion needs to be updated in System.IO.Pipelines/dir.props